x/gamm/keeper/multihop.go

```diff
@@ -79,14 +79,14 @@ func (k Keeper) createMultihopExpectedSwapOuts(ctx sdk.Context, routes []types.S
 			return nil, err
 		}
 
+		// fmt.Printf("calc in given out %s, pool liquidity %s\n", sdk.NewCoins(tokenOut), pool.GetTotalPoolLiquidity(ctx))
```

Comment:
```suggestion
```

Reply:
Thank you! Forgot to delete that debug code